src/com/lftechnology/java/training/srijan/FirstTutorial.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java.training.srijan;
+
+import java.util.Scanner;
+
+public class FirstTutorial {
+public static void main(String[] args){
+		
+		Scanner input = null;
+		
+		try{
+			input = new Scanner(System.in);
+			System.out.println("Enter your Name with Salutation");
+			String nameWithSalutation = input.nextLine();
+			nameWithSalutation=nameWithSalutation.trim();
+			System.out.printf("%n%s%n", nameWithSalutation);
+			
+			String[] text = nameWithSalutation.split(" ");
+			System.out.println("Salutation:" + text[0]);
+			//System.out.println("Name:" + text[1] +" "+ text[2]);
+			System.out.println("Name:");
+			for(int i=1;i<text.length;i++){
+				System.out.print(text[i] +" ");
+			}
+		}catch(Exception e){
+			System.out.println("There is an exception" + e);
```

Comment:
Add details for the exception. You should add the method where the exception occurred, the exception class and why it occurred. For example:
System.out.println("Inside" + FirstTutorial#main method +" while reading input from user. The exception class is: "+e.getClass.getSimpleName()+"Caused by : "e.getCause()+" Message is: "+e.getMessage() );